src/components/Table/GridTable.tsx

```diff
@@ -631,16 +631,88 @@ const VirtualRoot = memoizeOne<
   });
 });
 
-function calcGridColumns(columns: GridColumn<any>[], maxCardPadding: number | undefined): string {
-  let sizes = columns.map((c) => {
-    // Default to auto, but use `c.w` as a fr if numeric or else `c.w` as-if if a string
-    return typeof c.w === "string" ? c.w : c.w !== undefined ? `${c.w}fr` : "auto";
-  });
-  // If we're doing nested cards, we add extra 1st/last cells...
-  if (maxCardPadding) {
-    sizes = [`${maxCardPadding}px`, ...sizes, `${maxCardPadding}px`];
+/**
+ * Creates a `grid-template-column` specific to our virtual output.
+ *
+ * Because of two things:
+ *
+ * a) react-virtuoso puts the header in a different div than the normal rows, and
+ *
+ * b) content-aware sizing just in general look janky/constantly resize while scrolling
+ *
+ * When we're as=virtual, we change our default + enforce only fixed-sized units (% and px)
+ */
+export function calcVirtualGridColumns(columns: GridColumn<any>[], maxCardPadding: number | undefined): string {
```

Comment:
I think the challenge came when using `minmax`, `min-content`, `max-content` for the `w` attribute since both tracks couldn't talk to each other. I think here you just allow px, % and fr values.